docs/user-guide/config-management-plugins.md

```diff
@@ -231,6 +231,18 @@ If you don't need to set any environment variables, you can set an empty plugin
     is 90s. So if you increase the repo server timeout greater than 90s, be sure to set `ARGOCD_EXEC_TIMEOUT` on the
     sidecar.
 
+## Debugging a CMP
+
+If you are actively developing a sidecar-installed CMP, keep a few things in mind:
+
+1) If you are mounting plugin.yaml from a ConfigMap, you will have to restart the repo-server Pod so the plugin will
+   pick up the changes.
+2) If you have baked plugin.yaml into your image, you will have to build, push, and force a re-pull of that image on the
+   repo-server Pod so the plugin will pick up the changes. If you are using `:latest`, the Pod will always pull the new
+   image. If you're using a different, static tag, set `imagePullPolicy: Always` on the CMP's sidecar container.
+3) CMP errors are cached by the repo-server in Redis. Restarting the repo-server Pod will not clear the cache. Always
+   do a "Hard Refresh" when actively developing a CMP so you have the latest output.
```

Comment:
I initially assumed that the `--repo-cache-expiration` would also work for this cache, but it turned out not to be the case. Do you know if this is expected or if it would be worth adding a new one?

Reply:
Hm. That _ought_ to work. Do you have a test case showing that the manifests aren't expiring?